strings/traumas.json

```diff
@@ -33,7 +33,6 @@
         "closd for merbegging",
         "@pick(semicolon)pray can u @pick(create_verbs) @pick(create_nouns)???",
         "GEY AWAY FROM ME U GREIFING PRICK!!!!",
-        "ur a fuckeing autist!",
```

Comment:
i dont see why remove this out of everything in the file

Reply:
I am a autist whats wrong with this